placideirandora/questioner-with-node.js
# Conversion From ES5 To ES6

### What does this PR do?

It converts JavaScript code from ES5 to ES6.

### Description of Task to be completed

Existing JavaScript code will change from es5 to es6.

### How should this be manually tested?

You will not see any "**require**" and "**module.exports**" keywords which belong to "**es5**". Instead, You will see "**import**... **from**..." and "**export default**" which belongs to "**es6**".
